compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -487,13 +493,14 @@ function parseSimpleConstExpr(Tokenizer tok) returns SimpleConstExpr|err:Syntax
     return parseError(tok);
 }
 
-function parseOptQualIdentifier(Tokenizer tok, string identifier) returns [string?, string]|err:Syntax {
+function parseOptQualIdentifier(Tokenizer tok, string identifier) returns [string?, string, Position?]|err:Syntax {
```

Comment:
The first and third tuple members are both nil or both non-nil. The type doesn't capture that, which means using it is awkward. Should be `[[string, Position]?, string]`.

Reply:
fixed